### ISSUE TITLE
SampleLoopService::prepend_post_header() breaks

### ISSUE THREAD

**Issue by jbaldrich** [mwpd/basic-scaffold#2]
When passing an array of objects to `SimpleView::render()` the magic getter tries to escape an object and breaks. That's why SampleLoopService doesn't work, it should render the strings instead of the post object.

**Comment by robwoodgate** [comment]
Yes, string conversion has been [specifically excluded as a catchable error](https://bugs.php.net/bug.php?id=72948), so the try/catch doesn't work in the `SimpleView::get()` method, even with `\Throwable` catch type.

The code would work as-written, if `SimpleView` used a [runtime error handler](https://www.php.net/manual/en/class.errorexception.php#errorexception.examples) to make the error catchable.

However, I suspect the intention was just to escape strings and leave arrays, objects etc untouched... in which case, a simple check like so could work in `SimpleView::__get()`:

```
public function __get( $property ) {
    if ( array_key_exists( $property, $this->_context_ ) ) {
        $value = $this->_context_[ $property ];
        // Escape string values, otherwise return as-is
        return (is_string($value)) ? $this->escape( $value ) : $value;
    }
    ... rest of method ...
}
```

Otherwise, if the intention is to completely denature anything that is stringable (including objects implementing `__toString()`), then the calls to the $post object in the `test-loop-service.php` view need to change to use the `raw()` method like so:

 `$this->escape($this->raw('post')->post_title);`

Cheers
Rob

Ps - Deferring to @schlessera for the exact fix, but hope the above is useful to anyone playing with this cool boilerplate meantime.
